### ISSUE TITLE
Batch Normalization when training with Multi GPU settings 

### ISSUE THREAD

**Issue by shamanez** [yscacaca/DeepSense#15]
Hi I have modified this  code to run on multiple GPUs referencing [Cifar 10 multi GPU example](https://github.com/tensorflow/models/blob/master/tutorials/image/cifar10/cifar10_multi_gpu_train.py) .  But then I am not sure how to do batch normalization in distributed training . Specially 

Calculating moving averages for use in the inference . How to do this ?

**Comment by yscacaca** [comment]
Is it possible to calculate the moving averages on CPU? Such as `sess.run(mov_avg.assign(new_val))`

**Comment by shamanez** [comment]
possible. But then you have to get the statistics of one tower.
